pkg/kubectl/cmd/describe.go

```diff
@@ -95,6 +95,7 @@ func NewCmdDescribe(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmdutil.AddRecursiveFlag(cmd, &options.Recursive)
 	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on")
 	cmd.Flags().BoolVar(&describerSettings.ShowEvents, "show-events", true, "If true, display events related to the described object.")
+	cmd.Flags().BoolVar(&describerSettings.ShowAll, "show-all", false, "If true, display all hidden fields.")
```

Comment:
I don't think this flag adds much value; it only applies to nodes, and the help text is too vague to be meaningful. Let's just omit the conditions we don't care about; if you want to see all node conditions including false ones, you can still get them by running `kubectl get nodes $nodename -o yaml`.